### PR TITLE
Sanity connection for Cadre Team

### PR DESCRIPTION
**Pull request by mistyb01**
## What

> Summary of what you're changing.

Set up a Sanity schema to represent team members, so we can easily update the team on the Cadre page.

## Why Do

> The motivation behind your change.

To make the Cadre page a little easier to edit! As well as to pilot Sanity for this new iteration of the site.

## How Do

> Implementation choices, reviewer notes, caveats, etc.

Created a Sanity schema called TeamMember, along with a function to query the dataset for TeamMember objects. This function is then used to fetch data in `projects_cadre.tsx`!

Note that I left all the old Sanity files alone. I'm worried about messing with the currently live version of the page, so maybe after the rebrand gets published we can clean it up? But let me know if you have an opinion.

### CODE REVIEW

**Comment by santina** [comment]
> Note that I left all the old Sanity files alone. I'm worried about messing with the currently live version of the page, so maybe after the rebrand gets published we can clean it up? But let me know if you have an opinion.

Well you don't need to worry if it's being merged to rebranch :)

File: pages/project_cadre.tsx [hunk 1]
```diff
@@ -228,41 +240,15 @@ const ProjectIndividualPage = () => {
...
-              title="Jeffrey"
-              description="Program Manager"
-              image={Placeholder.src}
-            />
-            <CardWithPhoto
-              title="Jeffrey"
-              description="Program Manager"
-              image={Placeholder.src}
-            />
+            {teamData &&
```
Comment (santina) [inline pages/project_cadre.tsx:243]: nice!

File: pages/project_cadre.tsx [hunk 2]
```diff
@@ -1,15 +1,27 @@
...
-import Placeholder from 'assets/placeholder-person.png'
 import ProjectPlaceholder from 'assets/project-image.png'
-import CardWithPhoto from 'components/cards/CardWithPhoto'
 import StateBadge from 'components/cards/StateBadge'
 import { withBasicLayout } from 'components/layouts'
 import ListItemWithIcon from 'components/list/ListItemWithIcon'
+import CardWithPhoto from 'components/cards/CardWithPhoto'
 
 const ProjectIndividualPage = () => {
+  const [teamData, setTeamData] = useState(null)
```
Comment (santina) [inline pages/project_cadre.tsx:15]: Maybe just initialize with empty array? That way you can just do `teamData.map`

File: sanity/sanity-utils.ts [hunk 3]
```diff
@@ -0,0 +1,17 @@
+import { createClient, groq } from "next-sanity";
+import { apiVersion, dataset, projectId } from './env'
+
+export async function getTeamMembers(): Promise<TeamMember[]> {
+    const client = createClient({
+        projectId,
+        dataset,
+        apiVersion
+    })
+    return client.fetch(groq`*[_type == "team-member"]{
```
Comment (santina) [inline sanity/sanity-utils.ts:10]: I have no idea how sanity works, maybe for other people some comments on what each parameter and what this does would be helpful :)

**Comment by jnakaso** [comment]
Can we confirm that the dist folder needs to be added to the project?  Typically the list folder is generated during the build process.

**Review by gussmith23 — APPROVED** [review]
Cursory pass looks good to me, but I'm far from an export on any of these involved technologies. @jnakaso are you able to do a full pass?

**Review by jnakaso — APPROVED** [review]
This is great.  Thanks of updating the .gitignore

As an enhancement, I would add error handling & loading-progress indicator.

**Pull request merged**